### ISSUE TITLE
What distro do you use?

### ISSUE THREAD

**Issue by Vogeltak** [KeizerDev/.bashrc#1]
Did I see Debian?

**Comment by KeizerDev** [comment]
Nope, it's just Ubuntu 14.10 with a [Numix theme](https://github.com/numixproject/numix-icon-theme-circle). [This](http://satya164.deviantart.com/art/Numix-GTK3-theme-360223962) is pretty much how the windows looks. The only difference is that I don't use gnome but unity. And you're also a linux user? Do you use Manjaro?

**Comment by KeizerDev** [comment]
_Bump_

**Comment by Vogeltak** [comment]
Ah I'm sorry. I indeed use Manjaro, XFCE DE. But that is not the only distro I use. I also have a laptop with the net install of Manjaro where I set up bspwm and I also have a usb stick with an Antergos gnome install. As you can see, I quite like Arch based distributions. Too bad I haven't installed Arch yet though :P

The Pacman package manager is just so brilliant. I find apt quite devious, to be honest. But that may be because I haven't really researched it in great detail yet.

**Comment by KeizerDev** [comment]
That's cool! I've never worked with Arch based distributions. I've only used Ubuntu(on my laptop now), Linux Mint(for my old Windows Xp pc), Tails on my usb and sometime Fedora. I have some new pcs lying around, I may give Manjaro a try. It looks so cool with bspwm.  
I think apt is just the easiest package manager ever. Especially for linux starters, pacman is like `pacman -S gnome` and apt is like `apt-get install gnome` it's more human readable.

**Comment by Vogeltak** [comment]
Yeah go ahead and spin up a vm with [manjaro net install](http://sourceforge.net/projects/manjarolinux/files/release/0.8.12/netinstall/), it's really awesome and takes away some hassles which come with an Arch linux install. Prepare to read some documentation though.

Apt may be easier to read but it takes so long to type. Then there is also that unlogical way to search for packages `apt-cache search vim` or something like that, while in pacman it is just an extra 's' `pacman -Ss vim`.

**Comment by KeizerDev** [comment]
Yeah I will do! I'm only not a fan of a vm if I install it I will do it on a real machine immediately!

I totally agree with that, but if you want a package using apt-get I don't think there are a lot of people who use that to search. If you type something like `apt-get install gno` and you TAB you'll see the whole list of gnome dependencies. It's just that Ubuntu comes with apt-get...

**Comment by Vogeltak** [comment]
If you have a spare computer lying around you can use as a testbench, then that is what you should do! I too prefer a physical machine above a vm, but sadly I don't have an unlimited stack of them at my disposal.

**Comment by hayanderlandi** [comment]
Bash: /root/.bashrc: line 164: syntax error next to `token 'not expected` count'
Bash: /root/.bashrc: line 164: `if test (count $ argv) -eq 0 '

Ubuntu 16.10
